docs/pip_sphinxext.py

```diff
@@ -15,11 +15,15 @@
 
 class PipCommandUsage(rst.Directive):
     required_arguments = 1
+    optional_arguments = 4
 
     def run(self):
         cmd = create_command(self.arguments[0])
+        pip_cmd = '$ python -m pip'
+        if len(self.arguments) > 1:
+            pip_cmd = " ".join(self.arguments[1:])
```

Comment:
I feel it’d be better if we do

```rst
.. pip-command-usage:: install "python -m pip"
```

instead. The current argument format feels confusing since there’s no logic separation between the pip command (`install`) and the CLI prefix.

Also, I feel the `$` and `C:\>` prefix unnecessary here.

Reply:
I was not a fan as well. I added the quotes as suggested.